Makefile

```diff
@@ -576,18 +579,18 @@ govendor:
 .PHONY: fmt
 fmt: govendor genlocalfiles
 	@echo "+ $@"
-	@test -z "$$($(GOVENDOR) fmt +local)" || \
-		(echo "+ please format Go code with 'gofmt -s'" && /bin/false)
+	$(GOVENDOR) fmt +local || \
+		(echo "+ please format Go code with '$(GOFMT)'" && /bin/false)
```

Comment:
gofmt -s  only we would like the output

Reply:
this is just the message to the user; and user should overwrite source (rather than output to stdout):

```
[vagrant@dev skydive]$ gofmt --help
usage: gofmt [flags] [path ...]
  -cpuprofile string
        write cpu profile to this file
  -d    display diffs instead of rewriting files
  -e    report all errors (not just the first 10 on different lines)
  -l    list files whose formatting differs from gofmt's
  -r string
        rewrite rule (e.g., 'a[b:len(a)] -> a[b:]')
  -s    simplify code
  -w    write result to (source) file instead of stdout
``